product_classification_test_data/allposters/source.html

```diff
@@ -0,0 +1,2871 @@
0signal%20third%20party%20id')=1925474859338450424&amp;_cb_bt_data('user%20agent')=Mozilla%2F5.0%20(Macintosh%3B%20Intel%20Mac%20OS%20X%2010_12_3)%20AppleWebKit%2F602.4.8%20(KHTML%2C%20like%20Gecko)%20Version%2F10.0.3%20Safari%2F602.4.8&amp;_cb_bt_data('timestamp')=2017-07-23T19%3A15%3A12.464Z&amp;btpdb.PAsPhmI.dGZjLjUzMTMyNjk=U0VTU0lPTg&amp;btpdb.PAsPhmI.dWlkIC0gc2lnbmFsIGZpcnN0IHBhcnR5IGlk=Nzk1NzAwOTU5OTI4MDE2Mzg0MQ"></script><script src="https://js-agent.newrelic.com/nr-spa-1026.min.js"></script><script id="houzzwidget-js" async="" src="//platform.houzz.com/js/widgets.js?1500837285574"></script><script src="https://apis.google.com/_/scs/apps-static/_/js/k=oz.gapi.en_US.poyYqObhGtA.O/m=ratingbadge/exm=gapi_iframes/rt=j/sv=1/d=1/ed=1/am=AQ/rs=AGLTcCOWpFuuKrTFaxcYv_R5sIlu_oMvDA/cb=gapi.loaded_1" async=""></script><script src="https://apis.google.com/_/scs/apps-static/_/js/k=oz.gapi.en_US.poyYqObhGtA.O/m=gapi_iframes/rt=j/sv=1/d=1/ed=1/am=AQ/rs=AGLTcCOWpFuuKrTFaxcYv_R5sIlu_oMvDA/cb=gapi.loaded_0" async=""></script><script type="text/javascript" async="" src="https://www.google.com/_/scs/shopping-verified-reviews-static/_/js/k=boq-shopping-verified-reviews.VerifiedReviewsGcrBootstrapJs.en_US.y3Aby4Kp3N8.O/m=bootstrap/rt=j/d=1/rs=AC8lLkQbATGUfBlN4HT5RFtGZZs6EEV1JQ"></script><script type="text/javascript" src="//s.thebrighttag.com/tag?site=PAsPhmI&amp;docReferrer=https%3A%2F%2Fwww.google.com%2F&amp;H=38q0yjh"></script><script async="" src="//s.btstatic.com/tag.js#site=PAsPhmI"></script><script type="text/javascript" async="" src="http://www.google-analytics.com/analytics.js"></script><script type="text/javascript" async="" src="https://www.google-analytics.com/plugins/ua/ec.js"></script><script type="text/javascript" async="" src="//www.googleadservices.com/pagead/conversion_async.js"></script><script async="" src="//static.ads-twitter.com/uwt.js"></script><script async="" src="//px.airpr.com/airpr.js"></script><script src="//bat.bing.com/bat.js" async=""></script><script type="text/javascript" async="" src="http://www.googlecommerce.com/trustedstores/api/js"></script><script type="text/javascript" async="" src="//www.googleadservices.com/pagead/conversion_async.js"></script><script src="https://connect.facebook.net/signals/config/1220357044645971?v=2.7.18" async=""></script><script async="" src="https://connect.facebook.net/en_US/fbevents.js"></script><script async="" src="https://s.pinimg.com/ct/core.js"></script><style type="text/css">.pr-stars { background-image: url(/Reviews/pwr/engine/images/stars.gif); } .pr-stars-small { background-image: url(/Reviews/pwr/engine/images/stars_small.gif); }</style><script async="" src="https://www.google-analytics.com/analytics.js" id="scriptGA"></script><script async="" src="//www.googletagmanager.com/gtm.js?id=GTM-D84N"></script><script type="text/javascript" async="" src="//cdn3.optimizely.com/js/geo2.js"></script><script type="text/javascript" async="" src="//cdn3.optimizely.com/js/geo2.js"></script><title>
```

Comment:
I think I'm only collecting css for the nodes that have been loaded into the page (because in selenium, I'm getting all the nodes by calling document.getElementByTagName(*)), so these would match the source.html files right?